Add Foundation Information for OMSF (#7)
corsa-center/corsa-center.github.io: docs/foundations/omsf.md

---
layout: page
title: Open Molecular Software Foundation - How to Join
---

*February 2025*

*Zachary Baker*  

  
### **About OMSF**

The Open Molecular Software Foundation (OMSF) is a nonprofit organization that provides governance, infrastructure, and financial support for open-source projects in molecular sciences. We help researchers, developers, and industry partners build sustainable, high-impact software by offering operational resources, community engagement, and long-term strategic guidance.

> We work with academics looking for new ways to make their work sustainable, industries that want to promote open science, and community-based projects focused on events, education, and stewardship.

Reviewer: maybe rather than "academics" you could say "researchers"? To include researchers who are not in academic institutions, such national labs...

> We work with academics looking for new ways to make their work sustainable, industries that want to promote open science, and community-based projects focused on events, education, and stewardship.

Reviewer: ```suggestion
We work with researchers looking for new ways to make their work sustainable, industries that want to promote open science, and community-based projects focused on events, education, and stewardship.
```

---

### **Getting Started with OMSF**

The core of the Open Molecular Software Foundation is its hosted projects \- groups of talented people that drive innovation, foster collaboration, and contribute to our broader ecosystem.

> Hosting your project with OMSF means aligning with our commitment to open science and open-source principles while benefiting from an established infrastructure for funding management, governance, and community support.

Reviewer: ```suggestion
Hosting your project with OMSF means aligning with its commitment to open science and open-source principles while benefiting from an established infrastructure for funding management, governance, and community support.
```

To begin, consider:

* How your project fits within the computational molecular sciences domain.  
* Your commitment to open-source licensing and public code availability.  
* Your governance model and how decisions will be made within your project.  
* Your plan for financial sustainability, including potential funding sources.

> We are happy to work with you in structuring your proposal. If interested in hosting your project at OMSF, reach us at [info@omsf.io](mailto:info@omsf.io)

Reviewer: ```suggestion
OMSF is happy to work with you in structuring your proposal. If interested in hosting your project at OMSF, reach them at [info@omsf.io](mailto:info@omsf.io)
```

* Consider your community standards, such as a Code of Conduct.

**Governance and Structure:** 

* Define your project's leadership framework. Whether you have an advisory board, a technical leadership team, or a single project director, your governance model should ensure accountability and effective decision-making.   
* If you need help, OMSF can assist\! Reach out to [info@omsf.io](mailto:info@omsf.io) for guidance.

**Alignment:**

* Detail how your project aligns with OMSF’s commitment to the values of open science.  
* Outline your funding strategy \- whether through industry partnerships, government grants, or philanthropic support, tell us how you plan to sustain your project over time.  
* Consider how your project complements other OMSF-hosted projects to foster collaboration and shared success. Find that information here: [omsf.io/programs/projects/](http://omsf.io/programs/projects/)

**Planning and Development:** Clearly articulate your project's roadmap:

* What are your key milestones and deliverables?  
* What is your anticipated timeline for achieving them?  
* How will you measure progress and success over time?

> Do you not know the answer to some of these? That’s okay\! If you’d like guidance in how to answer these questions, reach out to [info@omsf.io](mailto:info@omsf.io), and we’d be happy to help.

Reviewer: ```suggestion
Do you not know the answer to some of these? That’s okay\! If you’d like guidance in how to answer these questions, reach out to [info@omsf.io](mailto:info@omsf.io), and OMSF will be happy to help.
```

1. **Initial Inquiry**: Prospective projects submit a proposal to [info@omsf.io](mailto:info@omsf.io)  
2. **Introductory Call**: A meeting between the project leads and OMSF representatives to discuss fit, expectations, and potential next steps.  
3. **Formal Application**: Submission of a detailed proposal covering governance, licensing, sustainability, and technical roadmap.  
4. **Review and Feedback**: OMSF evaluates the proposal, providing feedback and identifying areas for alignment or improvement.  
5. **Decision & Onboarding**: Approved projects receive an integration plan and guidance on leveraging OMSF’s resources.

---

### **What Happens in the Introductory Call?**

The introductory call serves as an opportunity for both OMSF and the prospective project to evaluate mutual alignment. Key topics covered include:

* The project’s mission, goals, and current status  
* The challenges the project faces and how OMSF can assist  
* Expectations for governance, infrastructure, and sustainability  
* The potential pathway for onboarding and next steps

> By the end, you will have a good idea if OMSF is the correct fit for you. Regardless, OMSF strives for an iterative process \- we want to see your project be the best that it can be\!

Reviewer: ```suggestion
By the end, you will have a good idea if OMSF is the correct fit for you. Regardless, OMSF strives for an iterative process \- it wants to see your project be the best that it can be\!
```

> Thanks for taking the time to read this guide. If you have any lingering questions, be sure to reach out to [info@omsf.io](mailto:info@omsf.io) \- we are more than happy to hear from you\!

Reviewer: ```suggestion
Thanks for taking the time to read this guide. If you have any lingering questions, be sure to reach out to [info@omsf.io](mailto:info@omsf.io) \- OMSF is more than happy to hear from you\!
```
